components/usage/pkg/apiv1/billing.go

```diff
@@ -368,6 +371,7 @@ func (s *BillingService) storeStripeCustomer(ctx context.Context, cus *stripe_ap
 	err := db.CreateStripeCustomer(ctx, s.conn, db.StripeCustomer{
 		StripeCustomerID: cus.ID,
 		AttributionID:    attributionID,
+		Currency:         cus.Metadata[stripe.PreferredCurrencyMetadataKey],
```

Comment:
Only new customers would have their Currency set on Stripe. Are we going to backfill this manually or are we gonna "live with it"?

Reply:
Definitely an option. Depending on the number of customers in stripe, it may also be easier to backfill manually.